.github/workflows/pythonpublish.yml

```diff
@@ -4,6 +4,8 @@
 name: Publish Python Package to PyPI
 
 on:
+  pull_request:     # delete this once workflow tested
+    branches: [main]  # delete this once workflow tested
```

Comment:
We also may have wanted to switch to using testpypi for testing the workflow

Reply:
ok, I'll do that.

Reply:
I've created an account on testpypi but the package isn't there for me to manage. How do I get it on testpypi for the very first time?

Reply:
Found it, doing it. https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives

Reply:
waiting for Arjun to give me permissions on the package in testpypi.